### DISCUSSION TITLE
Welcome to PlexPreferNonForcedSubs Discussions!

### DISCUSSION THREAD

**Discussion by RileyXX** [RileyXX/PlexPreferNonForcedSubs#16] [General]
## 👋 Welcome!

We’re using Discussions as a place to connect with other members of our community. We hope that you:
- Ask questions you’re wondering about.
- Share ideas.
- Engage with other community members.
- Welcome others and are open-minded. Remember that this is a community we build together 💪.

Before you post your Support question, have you tried solving the problem yourself:
1. Searching existing issues
2. Searching existing discussions

NOTE: Discussions are intended for community to community communication. Errors and Feature requests need to be reported as [issues](https://github.com/RileyXX/PlexPreferNonForcedSubs/issues/new/choose).

**Comment by MrSaiclops** [comment]
Would this tool work on users who were not hosting the library? I retrieved an auth token from inspect element (Since guests can't look at the XML of files) and I get this error: 
```cmd
--- ERROR ---
An error occurred while running the script.
Please submit the error to GitHub with the following information:
--------------------------------------------------
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/dist-packages/PlexPreferNonForcedSubs/PlexPreferNonForcedSubs.py", line 40, in main
    plex = PlexServer(baseurl, token)
  File "/usr/local/lib/python3.10/dist-packages/plexapi/server.py", line 116, in __init__
    data = self.query(self.key, timeout=self._timeout)
  File "/usr/local/lib/python3.10/dist-packages/plexapi/server.py", line 765, in query
    raise Unauthorized(message)
plexapi.exceptions.Unauthorized: (401) unauthorized; http://localhost:32400/ <html><head><script>window.location = window.location.href.match(/(^.+\/)[^\/]*$/)[1] + 'web/index.html';</script><title>Unauthorized</title></head><body><h1>401 Unauthorized</h1></body></html>

--------------------------------------------------
Submit the error here: https://github.com/RileyXX/PlexPreferNonForcedSubs/issues/new?template=bug_report.yml
```

Thanks!
